enh(release): Add Centreon Web 22.04.3 RN (#1754)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md

## Centreon Web

### 22.04.3

> 

Reviewer: ```suggestion

Release date: `August 25, 2022`

```

#### Enhancements

- [API] Added endpoint to perform all web updates
- [Authentication] Added a log message when an unregistered user tries logging in
- [Configuration] Use API to select metrics in virtual metrics configuration form
- [UI] Reduce spacing and align access buttons in user menu

#### Bug fixes

- [APIv1] Using the CLAPI import function no longer results in a PHP fatal error for the mentioned versions
- [Administration] Fixed consistency of ACLs with new poller creation wizard structure
- [Configuration] Fixed a regression: multiple trap definitions can use the same OID again

> - [Cron] Fixed SQL queries when databases names contain dash

Reviewer 2: ```suggestion
- [Cron] Fixed SQL queries when database names contain a dash
```

> - [Install] Allow to connect as centreon-engine user for Debian packaging

Reviewer 2: ```suggestion
- [Install] Make it possible to connect as user centreon-engine for Debian packaging
```

- [Install] Fixed Debian packages build when npm is not installed
- [Install] Fixed dependency name for Debian packaging
- [Monitoring] Fixed deletion of comments
- [Monitoring] Fixed the bug that canceled the display of text in graphics after an export in png

> - [UI] Fixed OpenId configuration form with Safari

Reviewer 2: ```suggestion
- [UI] Fixed OpenID configuration form with Safari
```

- [UI] Fixed dark mode theme switch
- [Widget] Fixed hostgroup multiple selection

#### Security fixes

- [Administration] Sanitized SQLi in media synchronization
- [Administration] Sanitized and bound ACL menus definitions queries
- [Administration] Sanitized and bound Auth class queries
- [Administration] Sanitized and bound queries in ACL actions definition

> - [Configuration] Fixed a XSS vulnerability in the broker configuration page

Reviewer 2: ```suggestion
- [Configuration] Fixed an XSS vulnerability in the Broker configuration page
```
